DuckDuckGo/NetworkProtectionDebugViewController.swift

```diff
@@ -95,19 +104,11 @@ final class NetworkProtectionDebugViewController: UITableViewController {
                 break
             }
 
+        case .debugFeature:
+            configure(cell, forDebugFeatureAtRow: indexPath.row)
+
         case .simulateFailure:
-            switch SimulateFailureRows(rawValue: indexPath.row) {
-            case .controllerFailure:
-                cell.textLabel?.text = "Enable NetP > Controller Failure"
-            case .tunnelFailure:
-                cell.textLabel?.text = "Enable NetP > Tunnel Failure"
-            case .crashFatalError:
-                cell.textLabel?.text = "Tunnel: Crash (Fatal Error)"
-            case .crashMemory:
-                cell.textLabel?.text = "Tunnel: Crash (CPU/Memory)"
-            case .none:
-                break
-            }
```

Comment:
I didn't want to do this change, but I was getting a cyclomatic complexity alert in this method.